### DISCUSSION TITLE
The argument can directly take a returned multiple values

### DISCUSSION THREAD

**Discussion by ghost** [vlang/v#14631] [Ideas]
for example, `gg.draw_rect_empty(int, int, int, int, gx.Color}` then..
```v
fn get_values() (int, int, int, int) {
	return 10, 10, 100, 100
}

struct Rect{
	x int = 10
	y int = 10
	w int = 100
	h int = 100
}

fn main(){
	draw_rect_empty(get_values(), gx.black)  // 1)
	draw_rect_empty(Rect{}, gx.black)        // 2)
}
```
It would be very convenient if this is accepted.

**Comment by dumblob** [comment]
This'd require a proper support for named/keyword arguments as proposed at the end of the discussion in https://github.com/vlang/v/pull/11135#issuecomment-924823890 .
